.gitignore

```diff
@@ -37,4 +37,4 @@ build/
 node_modules
 
 # Vagrant
-/tools/vagrant/.vagrant*
+.vagrant/
```

Comment:
What do you mean .vagrant/ is the hidden directory to save metadata even for multiple machines

```
.vagrant/machines/default/
.vagrant/machines/default/
```

The way I have it should ignore all directories '.vagrant/' anywhere

Can you provide a specific path you want to ignore?